doc/cephfs/snap-schedule.rst

```diff
@@ -162,6 +162,11 @@ Examples::
    snapshot creation is accounted for in the "created_count" field, which is a
    cumulative count of the total number of snapshots created so far.
 
+.. note: The maximum number of snapshots being retained is limited by the global
```

Comment:
The maximum number of snapshots to retain per directory....

Also, I would drop the word "global".

Reply:
* updated doc note
* added note in PendingReleaseNotes